apps/admin-gui/src/app/admin/pages/admin-page/admin-services/admin-services.component.html

```diff
@@ -2,6 +2,7 @@
   <h1 class="page-subtitle">{{'ADMIN.SERVICES.TITLE' | translate}}</h1>
   <perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
   <button
+    *ngIf="authResolver.isPerunAdmin()"
```

Comment:
on this page also put this if on the delete button please 
![image](https://user-images.githubusercontent.com/32871167/126630140-8f2907a8-995c-47f6-8bc4-859eae48414e.png)

Reply:
Fixed, thanks.